templates/kafka/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Apache Kafka
+description: |
+  (Experimental) Kafka cluster
+version: 0.8.2-rancher1
+category: clustering
+release by: Raul Sanchez
```

Comment:
I don't think you can have spaces in the keys, can you add an '_'? Also, this isn't a valid field for our API, though I don't think it hurts to have it there. I know we were toying with maintainer, but I don't see that in the code.